lib/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+
+class Book < Item
```

Comment:
As per the UML diagram provided to you, while creating a book, it should ask for `Genre`, `Author`, and `Label` details. However, in your case, it is only asking for the Label. Kindly fix this issue. A book belongs to an Item class and the item class has Genre, Author, and Label.